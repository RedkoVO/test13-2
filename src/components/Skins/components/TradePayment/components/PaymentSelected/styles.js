import CancelІSelected from '../../../../../../assets/images/svg/cancelІSelected.svg'

export default () => ({
  root: {
    width: '240px',
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'flex-end',
    paddingLeft: '30px',
    paddingRight: '20px',
    boxSizing: 'border-box'
  },
  paymentSelectedTitle: {
    fontSize: '12px',
    fontWeight: 500,
    alignSelf: 'flex-start',
    marginBottom: '7px'
  },
  paymentSelectedItem: {
    display: 'flex',
    width: '160px',
    paddingLeft: '5px',
    marginBottom: '2px',
    fontSize: '10px',
    fontWeight: 500,
    height: '11px',
    backgroundColor: 'rgba(0,0,0,0.13)',
    borderRadius: '5px'
  },
  selectedPrice: {
    fontSize: '10px',
    fontWeight: 500
  },
  selectedClose: {
    width: '8px',
    height: '8px',
    marginTop: '1px',
    marginLeft: '5px',
    background: `url(${CancelІSelected})`,
    backgroundRepeat: 'no-repeat',
    backgroundPosition: 'center',
    backgroundSize: 'contain'
  },
})
