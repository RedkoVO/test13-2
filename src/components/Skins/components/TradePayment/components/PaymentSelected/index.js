import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import styles from './styles'

const PaymentSelected = ({ classes, selectedItems }) => (
  <div className={classes.root}>
    <div className={classes.paymentSelectedTitle}>You selected 6 items:</div>
    {selectedItems.map((item, index) => (
      <div className={classes.paymentSelectedItem} key={item.id}>
        <div className={classes.selectedText}>
          {index}) {item.title} ....
        </div>
        <div className={classes.selectedPrice}>{item.cost}</div>
        <div className={classes.selectedClose} />
      </div>
    ))}
  </div>
)

PaymentSelected.propTypes = {
  classes: PropTypes.object,
  selectedItems: PropTypes.array
}

export default withStyles(styles)(PaymentSelected)
