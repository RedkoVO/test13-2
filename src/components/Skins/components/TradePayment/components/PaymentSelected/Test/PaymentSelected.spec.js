import * as React from 'react'
import renderer from 'react-test-renderer'

import PaymentSelected from '../'

const selectedItems = [
  { id: 1, title: 'Good EWeapon 1.2', cost: '$12.20' },
  { id: 2, title: 'Good EWeapon 1.2', cost: '$12.20' },
  { id: 3, title: 'Good EWeapon 1.2', cost: '$12.20' },
  { id: 4, title: 'Good EWeapon 1.2', cost: '$12.20' },
  { id: 5, title: 'Good EWeapon 1.2', cost: '$12.20' },
  { id: 6, title: 'Good EWeapon 1.2', cost: '$12.20' }
]

it('renders correctly', () => {
  const tree = renderer
    .create(<PaymentSelected selectedItems={selectedItems} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
