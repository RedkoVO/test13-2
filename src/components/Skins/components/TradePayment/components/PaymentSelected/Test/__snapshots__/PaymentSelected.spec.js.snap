// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="PaymentSelected-root-1"
>
  <div
    className="PaymentSelected-paymentSelectedTitle-2"
  >
    You selected 6 items:
  </div>
  <div
    className="PaymentSelected-paymentSelectedItem-3"
  >
    <div>
      0
      ) 
      Good EWeapon 1.2
       ....
    </div>
    <div
      className="PaymentSelected-selectedPrice-4"
    >
      $12.20
    </div>
    <div
      className="PaymentSelected-selectedClose-5"
    />
  </div>
  <div
    className="PaymentSelected-paymentSelectedItem-3"
  >
    <div>
      1
      ) 
      Good EWeapon 1.2
       ....
    </div>
    <div
      className="PaymentSelected-selectedPrice-4"
    >
      $12.20
    </div>
    <div
      className="PaymentSelected-selectedClose-5"
    />
  </div>
  <div
    className="PaymentSelected-paymentSelectedItem-3"
  >
    <div>
      2
      ) 
      Good EWeapon 1.2
       ....
    </div>
    <div
      className="PaymentSelected-selectedPrice-4"
    >
      $12.20
    </div>
    <div
      className="PaymentSelected-selectedClose-5"
    />
  </div>
  <div
    className="PaymentSelected-paymentSelectedItem-3"
  >
    <div>
      3
      ) 
      Good EWeapon 1.2
       ....
    </div>
    <div
      className="PaymentSelected-selectedPrice-4"
    >
      $12.20
    </div>
    <div
      className="PaymentSelected-selectedClose-5"
    />
  </div>
  <div
    className="PaymentSelected-paymentSelectedItem-3"
  >
    <div>
      4
      ) 
      Good EWeapon 1.2
       ....
    </div>
    <div
      className="PaymentSelected-selectedPrice-4"
    >
      $12.20
    </div>
    <div
      className="PaymentSelected-selectedClose-5"
    />
  </div>
  <div
    className="PaymentSelected-paymentSelectedItem-3"
  >
    <div>
      5
      ) 
      Good EWeapon 1.2
       ....
    </div>
    <div
      className="PaymentSelected-selectedPrice-4"
    >
      $12.20
    </div>
    <div
      className="PaymentSelected-selectedClose-5"
    />
  </div>
</div>
`;
